packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -100,8 +96,21 @@ async function handleSessionStream(
                 session.isSampled = isSampled(event.sessionUpdate)
             }
         }
-    } finally {
-        globals.clock.clearInterval(timer)
+    } catch (e) {
+        if (session.isAborted) {
+            //Expected case. User action cancelled stream (CodeLens, Close Editor, etc.).
+            //AbortSignal interrupts the LiveTail stream, causing error to be thrown here.
+            //Can assume that stopLiveTailSession() has already been called - AbortSignal is only
+            //exposed through that method.
+            getLogger().info(`Session ${session.uri.toString()} stopped.`)
```

Comment:
put variable info at the end of the message. please read the project guidelines, which mention this.
```suggestion
            getLogger().info(`Session stopped:  ${session.uri.toString()}`)
```